### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by jvanhoefer**

### CODE REVIEW

**Comment by jvanhoefer** [comment]
This should fix the issue with the missing tests for the SBML identifier/maths checks..., that @dilpath caught  in PR #122

**Comment by jvanhoefer** [comment]
Want to merge them before v022 sees the daylight ;)

**Review by dilpath — APPROVED** [review]
:+1:

File: tests/test_yaml2sbml.py [hunk 1]
```diff
@@ -76,6 +76,26 @@ def test_yaml_import_observables(self):
...
+        """
+        yaml_dir = os.path.join(self.test_folder,
+                                'ode_input_invalid_SBML_identifier.yaml')
+        with self.assertRaises(RuntimeError):
+            _parse_yaml(yaml_dir,
+                        'Test_Model')
+
+    def test_catch_invalid_math(self):
+        """
+        Test checks for strings, that parseL3Formula can not parse.
```
Comment (yannikschaelte) [inline tests/test_yaml2sbml.py:91]:
```suggestion
        Check for strings that parseL3Formula can not parse.
```
imperative / infinitive is usually preferred in docstrings

File: tests/test_yaml2sbml.py [hunk 2]
```diff
@@ -76,6 +76,26 @@ def test_yaml_import_observables(self):
 
         os.remove(sbml_test_dir)
 
+    def test_catch_invalid_sbml_identifier(self):
+        """
+        Test checks for invalid SBML identifiers.
```
Comment (yannikschaelte) [inline tests/test_yaml2sbml.py:81]:
```suggestion
        Check for invalid SBML identifiers.
```

**Review by yannikschaelte — APPROVED** [review]

**Pull request merged**